composer.json

```diff
@@ -6,12 +6,12 @@
     ],
     "require": {
         "php": ">=7.4.0",
-        "doctrine/annotations": "^1.8.0",
-        "doctrine/collections": "^1.6.8",
-        "doctrine/dbal": "^2.12.0",
+        "doctrine/annotations": "^1.13.3",
```

Comment:
Is there any particular reason we want to update these dependencies? This may block users of this library from upgrading for no reason.

Instead of this, I'd add a composer.lock file in order to make sure that we, as developers of the library, have new and locked versions 🤔

Reply:
I dont think there is a need for this now. Keep it as low as possible to support more wider variety of versions.